my_game.py

```diff
@@ -143,6 +147,11 @@ def __init__(self, **kwargs):
         # Pass arguments to class arcade.Sprite
         super().__init__(**kwargs)
         self.score = 0
+        self.lives = PLAYER_LIVES
+    def dies(self):
+        self.lives -= 1
+        if SOUND_ON:
+            Player.sound_dies.play()
```

Comment:
Please make the _die()_ function return _True_ if the _Player_ has less than 1 lives after the update. Return _False_ otherwise.